ui/public/locales/en.json

```diff
@@ -2440,6 +2442,7 @@
 "message.action.disable.zone": "Please confirm that you want to disable this zone.",
 "message.action.download.iso": "Please confirm that you want to download this ISO.",
 "message.action.download.template": "Please confirm that you want to download this Template.",
+"message.action.edit.nfs.options": "Please confirm that you want to change NFS mount options.<br>This will cause the storage pool to be remounted on all KVM hosts.",
```

Comment:
Do we also need to mention, changes will be in effect after cancelling the maintenance mode ?

Reply:
Mentioned now. Please check